packages/client-sdk-nodejs/package.json

```diff
@@ -16,8 +16,9 @@
     "prebuild": "eslint . --ext .ts",
     "test": "jest --testPathIgnorePatterns auth-client.test.ts",
     "integration-test-auth": "jest auth-client.test.ts",
+    "integration-test-vector": "jest vector-control-plane.test.ts",
```

Comment:
help me remember that i need to add vector-data-plane.test.ts to this when we merge my MVI pr